### ISSUE TITLE
The window always opens at the same location

### ISSUE THREAD

**Issue by aik** [Rob--W/open-in-browser#50]
https://github.com/Rob--W/open-in-browser/issues/7 is not fixed - the window opens on a laptop screen even though the external monitor is configured as primary and every other app honors that.

**Comment by aik** [comment]
is there a way to disable window positioning and let the system choose the location? thanks

**Comment by Rob--W** [comment]
> is there a way to disable window positioning and let the system choose the location?

No. [The logic responsible for moving the dialog](https://github.com/Rob--W/open-in-browser/blob/20e44a7576217121baa09f55c08f689654eabbd9/extension/dialog.js#L101) is unconditional.

Can you provide the exact steps to reproduce your problem? What OS and version are you using, window manager, display configuration, etc.

**Comment by aik** [comment]
Linux Fedora 27 or 29, firefox 64.0, lenovo thinkpad x1 laptop with intel graphics, gnome3 window manager 3.30.1, 2 displays - external 4K set as primary (gnome3 bars go there) and laptop fullhd screen as a secondary. The dialog always opens on the laptop screen no matter what. Here xrandr:

```
[aik@aik ~]$ xrandr --current | egrep "\+| connected"
eDP-1 connected 1920x1080+3840+631 (normal left inverted right x axis y axis) 309mm x 173mm
   1920x1080     60.05*+  60.01    59.97    59.96    59.93  
DP-2-2 connected primary 3840x2160+0+0 (normal left inverted right x axis y axis) 596mm x 335mm
   3840x2160     60.00*+  29.98  
```

**Comment by aik** [comment]
Things are actually better. When the addon is just installed, it works correctly - the window is on the main display. But if I disconnect the dock with external monitors, wait till the laptop screen becomes the main display, and then connect the dock back - the main display moves to the external monitor BUT firefox or addon (I bet it is firefox'es "screen" object) does not and therefore place the window to the laptop screen. I do not know how to resolve this :-/
